Dockerfile

```diff
@@ -97,32 +89,19 @@ USER tester
 RUN git config --global --add safe.directory '*'
 
 # Install terminus
-RUN curl -L https://github.com/pantheon-systems/terminus/releases/download/3.4.0/terminus.phar -o /usr/local/bin/terminus && \
+RUN curl -L https://github.com/pantheon-systems/terminus/releases/download/4.0.0/terminus.phar -o /usr/local/bin/terminus && \
     chmod +x /usr/local/bin/terminus
 RUN terminus self:update
 
-# Install CLU
-RUN mkdir -p /usr/local/share/clu
-RUN /usr/bin/env COMPOSER_BIN_DIR=/usr/local/bin composer -n --working-dir=/usr/local/share/clu require danielbachhuber/composer-lock-updater:^0.8.2
-
-# Install Drush
-RUN mkdir -p /usr/local/share/drush
-RUN /usr/bin/env composer -n --working-dir=/usr/local/share/drush require drush/drush "^10"
```

Comment:
Is drush not compatible with these PHP versions? I know version 10 is not great but I think any version works as a launcher, doesn't it? I think at this point every customer uses local drush so you just need something that works as a launcher

Reply:
Drush 10 was an odd choice, though; Drush 8 would also serve Drupal 7 users. D7 + Composer is rare, though.

I forgot about the launcher use case, since the platform doesn't need it. Since `vendor/bin` will always be in the same place, I could adjust the $PATH on the docker image to contain it; then `drush` would be available on the global $PATH, and we wouldn't need a launcher.

Reply:
`vendor/bin` actually isn't in the same location; this image can be used in a number of different CI providers, and also with bespoke customer-provided workflows, all of which may clone the Drupal site repository to any arbitrary location. We don't currently have a requirement to have `vendor/bin` in the $PATH, so I am going to restore Drush 10 to the image for b/c.

I'm not concerned about the Drush 8 oversight; that might have been the better choice in the past, but at this point, neither Pantheon nor Tag1 are supporting Composer-managed D7ES sites, so there is no need for Drush 8 here now.